### ISSUE TITLE
Missing accessibility labels for tags' task counters numbers in the sidebar

### ISSUE THREAD

**Issue by nekohayo** [getting-things-gnome/gtg#1365]
If you listen to what the Orca screen reader app "sees" when one of the tags is selected in the sidebar, you will hear:

> `Collapsed button` (instead of nothing, or something meaningful like "Tag:")  
> 
> `the tag name` (ex: "money").
> 
> `2.`

Where "2" at the end is the number of tasks, but without an explanation of what the "2" actually means. Ideally there should be some sort of accessibility label (or tooltip?) to make it so that the screen reader reads it as a translatable "2 tasks", or "2 tasks use this tag" or "2 tagged tasks" or something like that.

**Comment by nekohayo** [comment]
These problems are also hinted at when using the GTK Inspector's "Show accessibility warnings" global setting:

<img width="1050" height="479" alt="Image" src="https://github.com/user-attachments/assets/bb20b20e-2739-48e5-91b0-6d2c0855cadf" />
